0.download_data/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Runs processing related to step 0.download_data
+
+# convert all notebooks to python files into the nbconverted folder
+conda run -n jump_sc jupyter nbconvert --to python \
+    --output-dir=./0.download_data/nbconverted/ \
+    ./0.download_data/*.ipynb
+
+# download the jump manifest data
+# use a subshell to keep existing code intact
+(
+cd ./0.download_data || exit
```

Comment:
Could also use a trap for more flexibility in error handling

Reply:
Thanks good call, I think I'll leave this as-is because I'm not sure of any additional error handling we should perform at this time. Generally the subshell changes here came up as a result of [`shellcheck`'s SC2103](https://www.shellcheck.net/wiki/SC2103).

Reply:
This comment made me wonder if we should be careful about exceptions. As a result, I added `set -e` within the top of the file to help make sure we exit properly.